### ISSUE TITLE
iOS: dismiss keyboard, predictive covers key info

### ISSUE THREAD

**Issue by rdela** [silverbulletmd/silverbullet#377]
1.0(9)
0.2.12 version

I was going through the suggestions in the index and noticed when I tried to use the `Stats: Show` command that the output was covered up by the predictive text bar on top of the keyboard. There is a larger issue at play here which is it would be nice to be able to dismiss the keyboard easily, maybe with a button in the top row. I realize one can tap on an empty section of the page but that's not always easy nor obvious.

Tapping command:

![image](https://user-images.githubusercontent.com/74381/222576614-df78c9a8-93c6-4635-97d9-449c0032eff8.png)

Output obscured:

![image](https://user-images.githubusercontent.com/74381/222576715-dfe83fff-b367-4e16-acc8-3a8230272995.png)

**Comment by zefhemel** [comment]
Right, I've been looking at the CapacitorJS keyboard docs a bit, but there doesn't seem to be a lot of flexibility here. Currently the "resize" mode is set to native, which is supposed to resize the entire webview (UI) to fit above the keyboard. But indeed, it seems that doesn't take into account the auto suggest. Perhaps I should make a bug report there.

As to dismissing the keyboard: I've noticed this too. It's possible to programmatically dismiss the keyboard, but I'm not really sure when this is supposed to happen...

**Comment by zefhemel** [comment]
Closing this, because the native apps have now been replaced with the PWA.
